.scalafmt.conf

```diff
@@ -0,0 +1,22 @@
+// ScalaFmt stuff
+version = "2.6.1"
+
+# Exclude things in here
+project.excludeFilters = [.scalafmt.conf, "FunctionExecutionDispatcher.scala"]
+# Only format files tracked by git.
+project.git = true
+
+# change import a.b.{c,d} => import a.b.{ c, d }
+spaces.inImportCurlyBraces = true
+
+# vertically align function arguments with an arity of 2 or more
```

Comment:
I suggest updating to 3 or more. Opinions?

Reply:
I want all of them split for consistency sake, but I want others to weigh-in.

Reply:
yup, my motivation is primarily driven by maintenance of a reasonable line length and on average I would say that 2 parameters are still ok for a single line. 3 makes the line too long usually.